src/Symfony/Component/Serializer/Encoder/CsvEncoder.php

```diff
@@ -153,6 +153,10 @@ public function decode($data, $format, array $context = array())
         }
         fclose($handle);
 
+        if (isset($context['collection']) && true === $context['collection']) {
```

Comment:
Can be `if ($context['collection'] ?? false)` now that you target `master`.

Reply:
This should be covered by Coding Standard. Is that missed?

Reply:
This is not covered by CS atm I think, since fabbot is not red.